lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -2427,34 +2447,34 @@ int handle_csfb_s1ap_procedure_failure(
  **      Return:    RETURNok, RETURNerror                                  **
  **                                                                        **
  ***************************************************************************/
-
 int mme_app_notify_service_reject_to_nas(
```

Comment:
Return void and log message.
Make sure the caller of this function has appropriate logs